README.md

```diff
@@ -1,40 +1,40 @@
-DPimport: A command line glob importer for DPdash
-=================================================
+# DPimport: A command line glob importer for DPdash
+
 DPimport is a command line tool for importing files into DPdash using a
-simple [`glob`](https://en.wikipedia.org/wiki/Glob_(programming)) expression.
+simple [`glob`](<https://en.wikipedia.org/wiki/Glob_(programming)>) expression.
```

Comment:
Why does this need the `<>`s? Is it because of the parens in the url?

Reply:
There were pushes done to main so maybe this was done for a reason. I'm unsure what.